themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -163,7 +164,9 @@
           </#if>
         </div>
       </div>
-
+      <footer>
```

Comment:
Since this element is empty, perhaps we can simply omit it the tag for now? If folks want to add a footer element they can put it in the footer template.

Reply:
Thanks for pointing that out @jonkoops , yes we can give the user complete control of the fragment then.
Would you mind taking a look at the gist in https://github.com/keycloak/keycloak/pull/31391#issuecomment-2235836319 ?

I updated the gist and included your suggestion.